control-plane/pkg/reconciler/consumergroup/consumergroup.go

```diff
@@ -341,12 +347,30 @@ func (r Reconciler) schedule(ctx context.Context, cg *kafkainternals.ConsumerGro
 	return nil
 }
 
+func (r *Reconciler) maybeRequeueOnPendingReplicas(cg *kafkainternals.ConsumerGroup) {
+	// Instead of watching StatefulSet changes and globally resync all ConsumerGroups when the
+	// statefulset is scaled up or down, we requeue individual ConsumerGroups when the virtual
+	// replicas count doesn't match the expected total replicas, which means there are pending
+	// replicas to be scheduled and so the autoscaler will eventually scale up the statefulset
+	// to make room for _this_ consumer group.
+	//
+	// On scale down, the evictor is patching consumer groups, which means that we will reconcile
+	// the individual object that was affected by the eviction.
+	count := int32(0)
+	for _, p := range cg.Status.Placements {
+		count += p.VReplicas
+	}
+	if count != *cg.Spec.Replicas {
+		r.EnqueueAfter(cg, 5*time.Second)
```

Comment:
so we less or more reduce the "requeue" to a specific case, and avoid a constant (more frequent) global syncs ?